sapi/cli/php_cli_server.c

```diff
@@ -1231,6 +1292,13 @@ static php_socket_t php_network_listen_socket(const char *host, int *port, int s
 		}
 #endif
 
+#ifdef SO_REUSEPORT
+        {
```

Comment:
this should probably be conditional upon php_cli_server_workers_max, we probably don't want to reuse port unless actually forking to retain old behaviour ...